taskcluster/fenix_taskgraph/transforms/build.py

```diff
@@ -83,7 +84,7 @@ def add_nightly_version(config, tasks):
 
     for task in tasks:
         if task.pop("include-nightly-version", False):
-            task["run"]["gradlew"].append('-PversionName="{}"'.format(formated_date_time))
+            task["run"]["gradlew"].append('-PversionName={}'.format(formated_date_time))
```

Comment:
It turns out shell_quote handles spaces pretty well. So no need for extra double quotes, here.

https://github.com/mozilla-mobile/fenix/blob/3f41a4e9c63ffce45e4038b5f266613bccd88b56/taskcluster/fenix_taskgraph/job.py#L68